src/js/components/FormField/__tests__/__snapshots__/FormField-test.js.snap

```diff
@@ -761,7 +760,6 @@ exports[`FormField disabled with custom label 1`] = `
     >
       <label
         className="c2"
-        size="small"
```

Comment:
which node version are you using? make sure you are using ^12.18.

Reply:
Thanks. Updated node to 12.18.2
What's the issue with the review?

Reply:
The snapshots should not change, running `yarn test-update` on node 12.18 should fix the snapshots, can you update and push so the diff would point only to the relevant files?

Reply:
I'm confused. Don't these snapshots need to change since the whole point is FormField will no longer end up with size="small in it's DOM?

Reply:
From a quick look I've taken, I don't think that components such as Avatar, Sidebar, and Calendar should change their snapshots, unless you have a deeper understanding to share on those changes.

Reply:
Like FormField does, I believe they each use &lt;Text&gt; with a size prop. As a result, the DOM they produce will include a leaked size attribute. Therefore, their snapshots would also have size properties that need to go away as with FormField.

Reply:
Thanks for the insight, I'll need to dive a little into it to ensure that is what's happening.